### ISSUE TITLE
libgsl.so.0 error on RHEL6.9

### ISSUE THREAD

**Issue by durack1** [nco/nco#48]
@czender on an LLNL-built RHEL6.9 machine I got the following error:
```
$ yum install nco.x86_64
Loaded plugins: changelog, priorities, product-id, refresh-packagekit, rhnplugin, search-disabled-repos, security, subscription-manager,
              : versionlock
There was an error communicating with RHN.
RHN Satellite or RHN Classic support will be disabled.
rhn-plugin: Error communicating with server. The message was:
Unable to connect to the host and port specified
Setting up Install Process
Resolving Dependencies
--> Running transaction check
---> Package nco.x86_64 0:4.3.7-2.el6 will be installed
--> Processing Dependency: libnetcdf.so.6()(64bit) for package: nco-4.3.7-2.el6.x86_64
--> Processing Dependency: libgslcblas.so.0()(64bit) for package: nco-4.3.7-2.el6.x86_64
--> Processing Dependency: libgsl.so.0()(64bit) for package: nco-4.3.7-2.el6.x86_64
--> Running transaction check
---> Package nco.x86_64 0:4.3.7-2.el6 will be installed
--> Processing Dependency: libgslcblas.so.0()(64bit) for package: nco-4.3.7-2.el6.x86_64
--> Processing Dependency: libgsl.so.0()(64bit) for package: nco-4.3.7-2.el6.x86_64
---> Package netcdf.x86_64 0:4.1.1-3.el6.5 will be installed
--> Processing Dependency: libhdf5_hl.so.6()(64bit) for package: netcdf-4.1.1-3.el6.5.x86_64
--> Processing Dependency: libhdf5.so.6()(64bit) for package: netcdf-4.1.1-3.el6.5.x86_64
--> Running transaction check
---> Package hdf5.x86_64 0:1.8.5.patch1-10.el6 will be installed
---> Package nco.x86_64 0:4.3.7-2.el6 will be installed
--> Processing Dependency: libgslcblas.so.0()(64bit) for package: nco-4.3.7-2.el6.x86_64
--> Processing Dependency: libgsl.so.0()(64bit) for package: nco-4.3.7-2.el6.x86_64
--> Finished Dependency Resolution
Error: Package: nco-4.3.7-2.el6.x86_64 (epel)
           Requires: libgsl.so.0()(64bit)
Error: Package: nco-4.3.7-2.el6.x86_64 (epel)
           Requires: libgslcblas.so.0()(64bit)
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
```
Is there a work around to get this installed?

**Comment by czender** [comment]
4.3.7 is ~4 years old and I do not remember seeing this at the time and there's nothing in the ChangeLog that sparks a memory. I would run --skip-broken as suggested. Only ncap2 needs libgsl, so if it installs without libgsl you still get most of NCO. Other than that I guess I'd suggest installing the equivalent CentOS package and see if that works...Good luck!

**Comment by durack1** [comment]
@czender I've managed to jump forward in time to `v4.4.8`, circa April 2015.. And am still getting the same error:
```
ncrcat: error while loading shared libraries: libgsl.so.0: cannot open shared object file: No such file or directory
```
Is there a dependency that is no longer compatible with RHEL6.9? If I go through the process of upgrading to `v.4.6.5` will this likely fix it?

**Comment by czender** [comment]
Your guess is as good as mine. I havn't touched the yum/rpm build files in years, because Orion Poplawski and others  have kept them working flawlessly on Fedora, so I'm surprised RHEL is any different. GSL library numbering is tricky, though. You've got nothing to lose by trying to upgrade all the way to 4.6.5 except, of course, time :)

**Comment by durack1** [comment]
@czender I managed to get up-to-date to `nco 4.6.5` using the anaconda channel:
```
(env) user@machine:[~]:> conda install -c conda-forge nco
Fetching package metadata ...........
Solving package specifications: .
Package plan for installation in environment /home/user/anaconda2/envs/env:
The following NEW packages will be INSTALLED:
    nco: 4.6.5-0 conda-forge
```
However, I've now managed to get segfaults with my operations so will open a new issue for this
